src/controllers/sprite-details-controller.ts

```diff
@@ -0,0 +1,28 @@
+import { Controller } from "../controller.ts";
+
+export class SpriteCanvasData {
+  private _pixelSizePx: number = 32;
+  public get pixelSizePx(): number {
```

Comment:
I'm not a huge fan of this naming unless this is a convention for sprites that I don't know. I think, based on the code that this is supposed to be the size, in pixels, of the width of a sprite, but there's nothing about sprites or width in this name.

Reply:
Right, so.. This is the size that the canvas is representing the sprites pixels in... In pixels.

It doesn't say sprite in it because it doesn't actually have anything to do with the sprite, only the canvas' interpretation of the sprite when rendering.

I am happy to hear suggestions on a preferred name as I agree that it is not clearly describing what it is.. But I also couldn't think of a name that actually did describe it clearly.

Something like 'displayPixelSize' sounds like a boolean, 'canvasPixelSize' sounds like a width/height of the canvas in pixels, etc.

Maybe 'renderedPixelSize' or 'scaledPixelSize'? Dunno...

Reply:
This is just zoom, is it not?

Reply:
yep